doc/api/events.md

```diff
@@ -663,9 +663,13 @@ added: v11.13.0
 
 Creates a `Promise` that is resolved when the `EventEmitter` emits the given
 event or that is rejected when the `EventEmitter` emits `'error'`.
-The `Promise` will resolve with an array of all the arguments emitted to the
+The `Promise` will fulfill with an array of all the arguments emitted to the
 given event.
 
+Note: This method is intentionally generic and works with web platform
+[EventTarget](WHATWG-EventTarget) instances. `EventTarget`s have no special
+`'error'` event semantics and do not listen to the `'error'` event.
```

Comment:
Worth mentioning the different argument semantics? It resolves with the `Event` instance rather than an array of arguments.

Reply:
@apapirovski I was not sure about what to do here - what behavior would you think is less surprising?

Reply:
More saying we should make clear the distinction in arguments that the callback receives when you use this with `EventTarget`.